utils/obsproc/util.h

```diff
@@ -218,6 +218,24 @@ namespace gdasapp {
           oops::Log::test() << checksum(latitude_, "latitude") << std::endl;
           oops::Log::test() << checksum(datetime_, "datetime") << std::endl;
         }
+
+       // Swap the date
+       void reDate(int64_t minDAwindow, int64_t maxDAwindow, int windowcyc) {
+         for (int i = 0; i < location_; i++) {
+           if (datetime_(i) < minDAwindow) {
+             int timelatency = minDAwindow - datetime_(i);
```

Comment:
"timelatency" does not describe what that variable is ... Keep it simple! dt deltat deltatime, ... so your peers understand what it is you are trying to.

Reply:
Done.